drivers/python/tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+skipsdist = True
+envlist = py27, py34
```

Comment:
This list should include everything from Python2.6 through Python3.5. Adding `skip_missing_interpreters = True` would keep if from being a problem.